tests/frontend/test_kibana_app/cypress/integration/features/filters/add-new-filter.feature

```diff
@@ -4,9 +4,24 @@ Feature: Validate that the added filter label is displayed
     I want to set a new filter 
     in order to manage them
   
-   Scenario: The user add a new filer
+   Scenario Outline: The user add a new filer
```

Comment:
this scenario is something like "The user adds a new filter and it is deleted after navigation"
or something like that, right?

because in the test there is a validation that the filter is not present

Reply:
Hi @MauGaP, this scenario is related to the functionality to select a particular filter, and if you not pinned it the filter shouldnt persist if you move to the overview page or to another module.